fixtures/features/api-elements/x-example.sourcemap.json

```diff
@@ -400,7 +400,7 @@
                               "content": "application/x-www-form-urlencoded"
                             }
                           },
-                          "content": "id=false&greeting=hello"
+                          "content": "id=false&greeting=good%20morning"
```

Comment:
The example body generation is no longer respecting the `x-example` value and instead picking the default. The example value should take precedence.

Looking into the behaviour in the swagger adapter, we're abusing `enum` to do this. I think instead we can alter the code to use `default` to set the value to use.

Here's a patch for fury-adapter-swagger which will remove this change from this diff:

```diff
diff --git a/src/generator.js b/src/generator.js
index 5d22e5b..65ee5c6 100644
--- a/src/generator.js
+++ b/src/generator.js
@@ -69,9 +69,9 @@ export function bodyFromFormParameter(param, schema) {
   const retSchema = _.clone(schema);
 
   // If there's example value, we want to force the body generator
-  // to use it. This is done using 'enum' with a single value.
+  // to use it. This is done using 'default'
   if (param['x-example'] !== undefined) {
-    paramSchema.enum = [param['x-example']];
+    paramSchema.default = param['x-example'];
   }
 
   delete paramSchema.name;
```

@realityking Would you like to add the patch to your PR and re-generate these fixture. I think after the fixtures there is one set of files that is changed (`schema-reference.json`). The schema for this file is as follows:

```yaml
definitions:
  Item:
    type: object
    properties:
      id:
        type: number
        enum: [123]
        default: 456
      name:
        type: string
        default: Item name
    required: ['id', 'name']
```

The behaviour in the swagger adapter is causing the value to use the `default` over the first enum. Which I think makes sense.